[DOCS] New page: Detection & Response dashboard (#2085)
elastic/security-docs: docs/dashboards/overview-dashboard.asciidoc

[[overview-dashboard]]
= Overview dashboard

> The Overview dashboard provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation.

Reviewer: ```suggestion
The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation.
```

image::images/overview-pg.png[Overview dashboard]

[discrete]
== Live feed

> From the live feed on the Overview dashboard, you can quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.

Reviewer 2: ```suggestion
From the live feed on the Overview dashboard, you can quickly access recently created cases, favorited Timelines, and the latest {elastic-sec} news.
```

> From the live feed on the Overview dashboard, you can quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.

Reviewer: ```suggestion
The live feed on the Overview dashboard helps you quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.
```

> TIP: The *Security news* section provides the latest {elastic-sec} news to stay informed on new developments, learn about {elastic-sec} features, and more.

Reviewer: ```suggestion
TIP: The *Security news* section provides the latest {elastic-sec} news to help you stay informed of new developments, learn about {elastic-sec} features, and more.
```

image::images/live-feed-ov-page.png[Overview dashboard with live feed section highlighted]

[discrete]
== Histograms

> Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on a particular time, click and drag to select a time range, or choose a preset value. The *Stack by* dropdown lets you select which field is used to organize the data. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display alert counts by rule name within the specified time frame.

Reviewer 3: Per UX leads, we're supposed to use `menu` instead of `dropdown` (I personally like dropdown, but that was the consensus 😉 ) 
```suggestion
Time-based histograms show the number of detections, alerts, and events that have occurred within the selected time range. To focus on a particular time, click and drag to select a time range, or choose a preset value. The *Stack by* menu lets you select which field is used to organize the data. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display alert counts by rule name within the specified time frame.
```

> Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on areas of interest in time-based histograms, select a region to reflect a date range, or select a preset value in the timepicker. In the *Stack by* dropdown, you can select specific parameters to visualize individual counts. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display the total counts by alert name within the specified time frame.

Reviewer: ```suggestion
Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on a particular time, click and drag to select a date range, or choose a preset value. The *Stack by* dropdown lets you select which field is used to organize the data. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display alert counts by rule name within the specified time frame.
```

> TIP: All Elastic Security histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.

Reviewer 2: ```suggestion
TIP: All {elastic-sec} histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
```

Reviewer 4: I want to point out that the `inspect` functionality is not always there. For eg, the Cases chart, and table do not have the inspect functionality due to implementation details (cases are saved objects vs a query, and saved objects do not have associated queries...at least I think that's the case, @stephmilovic would be able to confirm) so we do not have the capability to show the inspect tool for these. IDK if that should change the content of what you have here, I think it's still fair, but just a heads up, just in case...

Reviewer: ^ so maybe just "many" instead of "all"

Author: ```suggestion
TIP: Many {elastic-sec} histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
```

[discrete]
== Host and network events

> View event and host counts specific to Elastic data shippers and apps, such as *Auditbeats* or *{endpoint-cloud-sec}*. Expand each category to view specific counts of hosts or network events related to the selected category.

Author: ```suggestion
View event and host counts grouped by data source, such as *Auditbeat* or *{endpoint-cloud-sec}*. Expand a category to view specific counts of host or network events related from the selected source.
```

> View event and host counts specific to Elastic data shippers and apps, such as *Auditbeats* or *Elastic Endpoint Security*. Expand each category to view specific counts of hosts or network events related to the selected category.

Reviewer 2: ```suggestion
View event and host counts specific to Elastic data shippers and apps, such as *Auditbeats* or *{endpoint-cloud-sec}*. Expand each category to view specific counts of hosts or network events related to the selected category.
```

Reviewer 2: Not sure if this is correct, so ignore if it's not!

Author: It's correct, and probably updated in another PR for the renaming of Endpoint Security, but might as well make sure it's updated here too.

> View event and host counts specific to Elastic data shippers and apps, such as *Auditbeats* or *Elastic Endpoint Security*. Expand each category to view specific counts of hosts or network events related to the selected category.

Reviewer: ```suggestion
View event and host counts grouped by data source, such as *Auditbeat* or *Elastic Endpoint Security*. Expand a category to view counts of host or network events from the selected source.
```

[role="screenshot"]
image::images/events-count.png[Host and network events on the Overview dashboard]

[discrete]
== Threat Intelligence

> The Threat Intelligence view on the Overview dashboard provides a streamlined way to collect threat intelligence data for threat detection and matching.

Reviewer: ```suggestion
The Threat Intelligence view on the Overview dashboard provides streamlined threat intelligence data for threat detection and matching.
```
